imports/client/components/CallSection.tsx

```diff
@@ -126,14 +126,14 @@ const ProducerManager = ({
   const producerServerCallback = useRef<ProducerCallback>();
 
   useEffect(() => {
-    if (producer && muted !== producer.paused) {
-      producer[muted ? 'pause' : 'resume']();
+    if (producer && paused !== producer.paused) {
+      producer[paused ? 'pause' : 'resume']();
```

Comment:
I seem to have overlooked this line entirely, which means my thesis was incorrect.  My apologies.

nit: can we make this a more standard function call structure?  The method-name-in-string-in-control-flow seems like it would be harder for tooling (or a human grepping about) to notice as a callsite of pause() or resume().

```
if (paused) {
  producer.pause();
} else {
  producer.resume();
}
```

Reply:
The tooling is actually totally fine with it :) but yes you're right, it's confusing, I'll switch